client/state/reader/feeds/selectors.js

```diff
@@ -17,7 +17,7 @@ export function shouldFeedBeFetched( state, feedId ) {
 function isStale( state, feedId ) {
 	const lastFetched = state.reader.feeds.lastFetched[ feedId ];
 	if ( ! lastFetched ) {
-		return false;
+		return true;
```

Comment:
yeah, just missed flipping the flag and we didn't have a test for "loaded, but no fetch time"